packages/infoportal-client/src/features/Meal/Pdm/MealPdmCashDashboard.tsx

```diff
@@ -100,75 +93,79 @@ export const MealPdmDashboard = () => {
                   dense
                   title={m.mealMonitoringPdm.treated}
                   data={data}
-                  filter={_ => _.feel_treated_respect === 'rcyc' || _.feel_treated_respect === 'rcmy'}
+                  filter={_ => _.answers.feel_treated_respect === 'rcyc' || _.answers.feel_treated_respect === 'rcmy'}
                 />
                 <ChartPieWidgetBy
                   dense
                   title={m.mealMonitoringPdm.provideInfo}
                   data={data}
-                  filter={_ => _.organization_provide_information === 'yes'}
+                  filter={_ => _.answers.organization_provide_information === 'yes'}
                 />
                 <ChartPieWidgetBy
                   dense
                   title={m.mealMonitoringPdm.satisfiedProcess}
                   data={data}
-                  filter={_ => _.satisfied_process === 'ndyl' || _.satisfied_process === 'ndna'}
+                  filter={_ => _.answers.satisfied_process === 'ndyl' || _.answers.satisfied_process === 'ndna'}
                 />
                 <ChartPieWidgetBy
                   dense
                   title={m.mealMonitoringPdm.correspond}
                   data={data}
-                  filter={_ => _.amount_cash_received_correspond === 'yes'}
+                  filter={_ => _.answers.amount_cash_received_correspond === 'yes'}
                 />
                 <ChartPieWidgetBy
                   dense
                   title={m.mealMonitoringPdm.problems}
                   data={data}
-                  filter={_ => _.experience_problems === 'yes'}
+                  filter={_ => _.answers.experience_problems === 'yes'}
                 />
               </SlidePanel>
               <Panel savableAsImg expendable title={m.location}>
                 <PanelBody>
-                <MapSvgByOblast
-                  sx={{maxWidth: 480, margin: 'auto'}}
-                  fillBaseOn="value"
-                  data={data}
-                  getOblast={_ => mapOblast[_.ben_det_oblast!]}
-                  value={_ => true}
-                  base={_ => _.ben_det_oblast !== undefined}
-                />
+                  <MapSvgByOblast
+                    sx={{maxWidth: 480, margin: 'auto'}}
+                    fillBaseOn="value"
+                    data={data}
+                    getOblast={_ => mapOblast[_.answers.ben_det_oblast!]}
+                    value={_ => true}
+                    base={_ => _.answers.ben_det_oblast !== undefined}
+                  />
                 </PanelBody>
               </Panel>
               <Panel title={m.ageGroup}>
                 <PanelBody>
-                  <AgeGroupTable tableId="pdm-dashboard" persons={data.flatMap(_ => _.persons)} enableDisplacementStatusFilter enablePwdFilter/>
+                  <AgeGroupTable tableId="pdm-dashboard" persons={ctx.fetcherAnswers.get?.flatMap(_ => _.persons)} enableDisplacementStatusFilter enablePwdFilter/>
```

Comment:
It's confusing to juggle between `ctx.fetcherAnswers.get` and `data`. If you need to access persons, probably better to add it in `PdmData`. The idea is to make `PdmData` the bottleneck of all this module. So devs don't have to guess from which montain the source is coming.